updating/updating-restricted-network-cluster.adoc

```diff
@@ -38,11 +38,23 @@ include::modules/cli-installing-cli.adoc[leveloffset=+3]
 
 // this file doesn't exist, so I'm including the one that should pick up more changes from Clayton's PR - modules/installation-adding-mirror-registry-pull-secret.adoc[leveloffset=+1]
 
-include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+2]
+include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+3]
 
-include::modules/update-mirror-repository.adoc[leveloffset=+2]
+[id="update-mirror-repository_{context}"]
+=== Mirroring the {product-title} image repository
 
-include::modules/machine-health-checks-pausing.adoc[leveloffset=+3]
+Before you update a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+include::modules/update-mirror-repository-oc-mirror.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../installing/disconnected_install/installing-mirroring-disconnected.adoc[Mirroring images for a disconnected installation using the oc-mirror plug-in]
```

Comment:
Good job making the extension .adoc, but you also need to add the anchor after it.

```suggestion
* xref:../installing/disconnected_install/installing-mirroring-disconnected.adoc#installing-mirroring-disconnected[Mirroring images for a disconnected installation using the oc-mirror plug-in]
```

FYI in case you aren't aware, don't click the "Commit suggestion" button that appears when someone does a suggested change like how I did here ^